tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -12,6 +12,11 @@
 from great_expectations.rule_based_profiler.types import DataAssistantResult, Domain
 from great_expectations.util import deep_filter_properties_iterable
 
+try:
+    from unittest import mock
+except ImportError:
+    from unittest import mock
+
```

Comment:
I haven't seen this "try it again if it didn't work the first time" pattern before in code, only in real life 😂  Joking aside, am I missing something?

Reply:
No you aren't missing anything. I copied this from elsewhere...it is sprinkled all over the place.

Reply:
😮